__tests__/app.test.js

```diff
@@ -325,3 +325,30 @@ describe("PATCH /api/articles/:article_id", () => {
       })
   })
 })
+
+describe("DELETE /api/comments/:comment_id", () => {
+  test("204: deletes the given comment by comment_id and responds with no content", () => {
+    return request(app)
+      .delete("/api/comments/1")
+      .expect(204)
+      .then(({ body }) => {
+        expect(body).toEqual({})
+      })
```

Comment:
This .then(...) isnt needed as 204 always results in an empty object and this is just checking express works as it should which isnt needed